db/schema.rb

```diff
@@ -330,6 +329,7 @@
     t.datetime "refresh_started_at"
     t.datetime "refresh_finished_at"
     t.datetime "last_successful_refresh_at"
+    t.jsonb "info"
```

Comment:
@hsong-rh Nice this will allow us to add information about the Tower that we can display to the end user. So they would know which version of the tower this is. We have the following info today for now we are only going to store 
```
 "version": "3.5.0",
 "ansible_version": "2.8.0"
```

This data is coming from the /api/v2/config/ REST API call